src/plugins/calibration/calibration.cpp

```diff
@@ -30,6 +30,11 @@ void Calibration::calibrate_magnetometer_async(CalibrateMagnetometerCallback cal
     _impl->calibrate_magnetometer_async(callback);
 }
 
+void Calibration::calibrate_levelhorizon_async(CalibrateLevelHorizonCallback callback)
```

Comment:
Did that get generated like this? I think it should be `calibrate_level_horizon_async` :thinking:

Reply:
Ha, same comment :)

Reply:
Probably because I had that in place already. Let me check

Reply:
But then it wouldn't compile, right?